teachopencadd/talktorials/T014_binding_site_detection/talktorial.ipynb

```diff
@@ -0,0 +1,2116 @@
+{
```

Comment:
<p>SURFNET: "...any two pairs of atoms on the protein surface directly." Meaning atoms must be neighbors or really any two atoms on the surface?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>I think yes (see below), anyhow actually the sentence will be changed to: *all pairs of atoms*</p><blockquote>SURFNET differs in that it considers all relevant pairs of atoms in turn and places a sphere midway between each pair, reducing its size if it clashes with any neighboring atoms	</blockquote><p>And later for a single molecule:</p><blockquote>...&nbsp;Here all pairs of atoms from the one molecule are considered.</blockquote>

Reply:
<p>Done.</p>

Reply:
<p>As far as I remember they don't say anything about a cut-off explicitly, but there might still be some <em>trick</em> otherwise it might be computationally very expensive.</p><p> </p><p>For the second point, yes, exactly those spheres will <em>disappear</em>.  </p>